sentry/src/test/java/io/sentry/HubTest.kt

```diff
@@ -314,7 +314,7 @@ class HubTest {
         val exception = RuntimeException()
         val span = mock<Span>()
         whenever(span.spanContext).thenReturn(SpanContext("op"))
-        sut.setSpanContext(exception, span)
+        sut.setSpanContext(exception, io.sentry.util.Pair(span, "tx-name"))
```

Comment:
import the class instead full package

Reply:
Kotlin Pair is already used in this class.

Reply:
we can still import and use an alias

`import kotlin.test.assertEquals as hi`